types/react-relay/index.d.ts

```diff
@@ -4,6 +4,7 @@
 //                 Matt Martin <https://github.com/voxmatt>
 //                 Eloy Durán <https://github.com/alloy>
 //                 Nicolas Pirotte <https://github.com/npirotte>
+//                 Cameron Knight <https://github.com/ckknight>
```

Comment:
Are you aware that this means signing up to be a maintainer and thus being pinged for reviews on future PRs?

Reply:
+1

Reply:
I'm comfortable with this.

Reply:
Ace 👌